spec/migrations/20180424141617_azure_backslash_to_forward_slash_spec.rb

```diff
@@ -0,0 +1,116 @@
+require_migration
+
+describe AzureBackslashToForwardSlash do
+  let(:vm_stub) { migration_stub :Vm }
+  let(:stack_stub) { migration_stub :OrchestrationStack }
+  let(:stack_output_stub) { migration_stub :OrchestrationStackOutput }
+  let(:stack_param_stub) { migration_stub :OrchestrationStackParameter }
+  let(:stack_resource_stub) { migration_stub :OrchestrationStackResource }
+  let(:event_stream_stub) { migration_stub :EventStream }
+
+  let!(:vm) do
+    vm_stub.create!(
+      :name        => 'azure_test_vm',
+      :ems_ref     => 'xyz\some_group\microsoft.compute/virtualmachines\foo',
+      :uid_ems     => 'xyz\some_group\microsoft.compute/virtualmachines\foo',
+      :type        => 'ManageIQ::Providers::Azure::CloudManager::Vm',
+      :description => 'some\description'
+    )
+  end
+
+  let!(:template) do
+    vm_stub.create!(
+      :name        => 'azure_test_template',
+      :ems_ref     => 'xyz\some_group\microsoft.compute/virtualmachines\bar',
+      :type        => 'ManageIQ::Providers::Azure::CloudManager::Template',
+      :description => 'some\other\description'
+    )
+  end
+
+  let!(:stack) do
+    stack_stub.create!(
+      :id      => 1,
```

Comment:
Do not hardcode ids in tests as it circumvents the ability to test with a random region id.